File: app/src/main/java/com/iuicity/xinjr/feature/login/LoginConstract.java
package com.iuicity.xinjr.feature.login;

import com.iuicity.xinjr.base.IBaseView;

/**
 * Created by Shui on 2017/7/5.
 */

interface LoginConstract {
    interface View extends IBaseView {
        void gotoMain();
    }

    interface Presenter {
        void login(String account, String pwd);
    }
}

File: app/src/main/java/com/iuicity/xinjr/feature/login/LoginConstract.kt
package com.iuicity.xinjr.feature.login

import com.iuicity.xinjr.base.IBaseView

/**
 * Created by Shui on 2017/7/5.
 */

interface LoginConstract {
    interface View : IBaseView {
        fun gotoMain()
    }

    interface Presenter {
        fun login(account: String, pwd: String)
    }
}
